.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,35 @@
+name: Test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
I think we should go for `requirements-test.txt` instead, as this dependency don't actually have any requirements that's not specified in `setup.py` (which is Django above version 2.2). Can be merged for now, but has to be looked at more when solving #2. 
My intended folder structure will be:
```python
django-guid git:(master) tree                 
.
├── CHANGELOG.rst
├── django_guid
│   ├── __init__.py
│   ├── config.py
│   ├── log_filters.py
│   ├── middleware.py
├── tests
│   ├── __init__.py
│   ├── test_files.py
├── README.rst
├── setup.py
``` 
Which means the test requirements will most likely also be under the `tests` folder.

Reply:
Changed to `requirements-test.txt` and put it inside `/tests/` module.